src/ZombieDriver.php

```diff
@@ -94,13 +94,14 @@ public function stop()
     public function reset()
     {
         $js = <<<JS
-browser.destroy();
-browser = null;
```

Comment:
Well, you were the one saying there is a problem in case the window is switched. And MinkExtension indeed does not do anything to handle it.

the fact that nobody complained can have 2 reasons:

- your initial assumption that there is an issue on reset when a popup is opened is wrong
- nobody is using popups with Mink

This is why I asked to write a test proving your point that there is an issue, so that we don't depend on usage.